examples/keycloak/myubuntu.id.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: myubuntuid
+  name: myubuntuid
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: myubuntuid
+  template:
+    metadata:
+      annotations:
+        admission.trusted.identity/inject: "true"
+        # token-url: complete URL for obtaining a realm token:
+        tsi.identities: |
+          - tsi.keycloak/token-url: "http://keycloak.server/auth/realms/tsi-authz/protocol/openid-connect/token"
+            tsi.keycloak/local-path: "tsi-secrets/identities"
```

Comment:
we need to add another field here called:

tsi.keycloak/audiences, which will contain one or more audiences of which to create tokens for. Could be comma delemetered.